presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -590,6 +590,34 @@ Statistical Aggregate Functions
     Returns linear regression slope of input values. ``y`` is the dependent
     value. ``x`` is the independent value.
 
+.. function:: regr_avgx(y, x) -> double
+    Returns the average of the independent value in a group. ``y`` is the dependent
+    value. ``x`` is the independent value.
+
+.. function:: regr_avgy(y, x) -> double
+    Returns the average of the dependent value in a group. ``y`` is the dependent
+    value. ``x`` is the independent value.
+
+.. function:: regr_count(y, x) -> double
+    Returns the number of non-null pairs of input values. ``y`` is the dependent
+    value. ``x`` is the independent value.
+
+.. function:: regr_r2(y, x) -> double
+    Returns the coefficient of determination of the linear regression. ``y`` is the dependent
+    value. ``x`` is the independent value.
+
+.. function:: regr_sxy(y, x) -> double
+    Returns the sum of the product of the dependent and independent values in a group. ``y`` is the dependent
+    value. ``x`` is the independent value.
+
+.. function:: regr_syy(y, x) -> double
+    Returns the sum of the squares of the dependent values in a group. ``y`` is the dependent
+    value. ``x`` is the independent value.
+
+.. function:: regr_sxx(y, x) -> double
```

Comment:
```suggestion
.. function:: regr_sxx(y, x) -> double

```

Reply:
Thanks, Updated. Here's the new screenshot of what the doc build looks like:
![image](https://github.com/prestodb/presto/assets/31331967/9a7de004-3596-4b63-9874-98ac3e841fc1)

Reply:
> Thanks, Updated. Here's the new screenshot of what the doc build looks like: 

Thanks! I just now did a new pull and local build of the doc and everything looks good! Updating my review to Approve for the docs.